Maddori.Apple/Maddori.Apple/Screen/Home/TeamDetail/TeamDetailViewController.swift

```diff
@@ -37,34 +37,19 @@ final class TeamDetailViewController: BaseViewController {
         label.font = .label2
         return label
     }()
-    private let profileImageView: UIImageView = {
-        let imageView = UIImageView()
-        imageView.layer.cornerRadius = 23
-        imageView.image = ImageLiterals.imgDefaultProfile
-        return imageView
-    }()
-    private let profileNicknameLabel: UILabel = {
-        let label = UILabel()
-        // FIXME: - API 연결 후 삭제
-        label.text = "이두"
-        label.font = .label2
-        label.textColor = .gray600
-        return label
-    }()
-    private let profileRoleLabel: UILabel = {
-        let label = UILabel()
-        // FIXME: - API 연결 후 삭제
-        label.text = "디자인 리드 / 개발자"
-        label.font = .caption2
-        label.textColor = .gray400
-        return label
-    }()
+    private let memberInformationView = MemberInformationView(nickname: "이드", role: "디자인 리드 / 개발자")
     private let dividerView: UIView = {
         let view = UIView()
         view.layer.borderWidth = 0.5
```

Comment:
p1;
```suggestion
        view.layer.borderWidth = 1
```
앗 이거 1인 것 같아여!

Reply:
@Guel-git 요게 아마 위 아래 0.5 씩 두개라 1이 되도록 만든..! 걸거입니다..!

Reply:
앗 그래요? 아 밑에서도 1로 height 를 잡네요 ㅎㅎ! 오호라 상관없네요!

Reply:
맞습니다!
호야 말대로 위아래 두개해서 0.5 + 0.5해서 1로 했습니다!
꼼꼼한 리뷰 좋아요~~ ^_______^